dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/FileUtils.java

```diff
@@ -244,4 +244,15 @@ public static boolean directoryTraversal(String filename){
         }
     }
 
+    /**
+     * Get file size in KB
+     *
+     * @param filename String type of filename
+     * @return file size in KB
+     */
+    public static double getFileSizeInKB(String filename){
+        File file = new File(filename);
+        return ((double) file.length()) / 1024;
```

Comment:
how about add 1024 to constants?

Reply:
Sure, thx. Will fix it.

Reply:
You can directly use org.apache.commons.io.FileUtils.ONE_KB

Reply:
@ruanwenjun `org.apache.commons.io.FileUtils.ONE_KB` is cool! Thx. I will fix it in the next commit.

Reply:
Fixed in by the latest commit.